### PR TITLE
Add energyintegral hmu

### PR DESCRIPTION
**Pull request by dabbelju007**
Hi @john30,

this is my first pull request. I hope I did everything right. I did add a new value "EnergyIntergal" to 08.hmu.csv in Vaillant de and en.

Tested on my aroTHERM plus VWZ MEH 97/6.

Thanks
Dabbelju

### CODE REVIEW

**Comment by peterpickle** [comment]
I tried this with an aroTherm 85/3 but I get the error below when doing "ebusctl find -r".

hmu EnergyIntegral =  (ERR: invalid position for 3108b51a0405ff3221 / 02ff01)

My devices an config:
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU01;SW=0306;HW=8802", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=70000;SW=0614;HW=6903", loaded "vaillant/15.700.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZIO;SW=0111;HW=0103"
address e8: slave, scanned "MF=Vaillant;ID=FMU00;SW=0204;HW=6502"

**Comment by dabbelju007** [comment]
Hi @peterpickle,

thanks for your feedback. First of all here my devices and config:

`address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0607;HW=5103", loaded "vaillant/08.hmu.csv"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=72000;SW=0122;HW=7703", loaded "vaillant/15.720.csv"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZIO;SW=0607;HW=5103"
address f1: master #10
address f6: slave #10, scanned "MF=Vaillant;ID=NETX2;SW=4025;HW=5703"`

I am not sure about the different HMU (00 and 01). Perhaps at your HMU the value is stored in a different place.

Do other values like CurrentYieldPower, CurrentConsumedPower or CurrentCompressorUtil work for you?

**Comment by peterpickle** [comment]
Hi @dabbelju007,

I just checked. They seem to work fine.

```
hmu CurrentCompressorUtil = 46
hmu CurrentConsumedPower = 1.5
hmu CurrentYieldPower = 3.9
```
I do seem to get a reply message back (02ff01) so size 2, and 0xFF01 as value. 
I'm not sure what to expect as a value.
What value does your system return?
Do you know how many bytes should get returned?

File: ebusd-2.1.x/en/vaillant/08.hmu.csv [hunk 1]
```diff
@@ -9,6 +9,7 @@ r,,Hours,,,,,40,,,energy,,,,,,,,,,,,,,,,,,,,,
 r,,HoursHc,,,,,41,,,energy,,,,,,,,,,,,,,,,,,,,,
 r,,HoursCool,,,,,43,,,energy,,,,,,,,,,,,,,,,,,,,,
 # some more statistics from Live Monitor,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+r,,EnergyIntegral,,,,,21,,,integral,,°min,aktuelles Energieintegral in °min,,,,,,,,,,,,,,,,,,
```
Comment (john30) [inline ebusd-2.1.x/en/vaillant/08.hmu.csv:12]: should be english comment

**Review by john30 — CHANGES_REQUESTED** [review]

**Comment by dabbelju007** [comment]
Hi John,

its translated.

Thanks
Dabbelju

**Review by john30 — APPROVED** [review]

**Pull request merged**